corehq/apps/app_manager/templates/app_manager/v1/partials/form_filter.html

```diff
@@ -10,7 +10,7 @@
     </label>
     <div id="form-filter" class="col-sm-4 row">
         <div data-bind="css: {'has-error': !allowed()}">
-            <textarea name="form_filter" class="form-control vertical-resize" data-bind="
+            <textarea name="form_filter" class="form-control" data-bind="
```

Comment:
took a second look, this is already taken care of, because form filter uses the xpath validator, which turns off spellcheck.